src/charm.py

```diff
@@ -1313,6 +1313,7 @@ def update_config(self, is_creating_backup: bool = False) -> bool:
             backup_id=self.app_peer_data.get("restoring-backup"),
             stanza=self.app_peer_data.get("stanza"),
             restore_stanza=self.app_peer_data.get("restore-stanza"),
+            parameters=self.postgresql.build_postgresql_parameters(self.config["profile"], 0),
```

Comment:
Should the user be able to change the profile after the charm was already deployed?

Reply:
Yes - it will need to happen - to keep it simple I will do on follow up PR.